### DISCUSSION TITLE
Language support

### DISCUSSION THREAD

**Discussion by jymden** [statamic-rad-pack/runway#177] [Feature Requests]
I should be rather easy to implement language support by language keys in in the blades and vue components.

**Comment by duncanmcclean** [comment]
I'd accept a PR for it 😄

**Comment by jymden** [comment]
If I have the time to do that, I definitively will! Thank you so much for our great work. Runway really saved me so much work today.

**Reply by duncanmcclean** [reply]
Awesome! I might have some time this week to do it but any actual translations will obviously need to be PR'd. I only know English 😆

**Comment by duncanmcclean** [comment]
Cool, I've made it so all of the strings Runway outputs in the Control Panel should be translatable (#178). 🎉 

Happy to accept translations for any languages - no pressure of course!

**Reply by jymden** [reply]
Fantastic! Some of them popped right in since the keys are the same as in base Laravel/Statamic. I'm quite new att working with PR's for composer packages. But I might have a go at it with this one. I think something is not right with the vue-components though. It seems as they just output the whole key. I looked into how Statamic did this and they wrapped it in {{ }} so blade templating takes over in the html. I tried that but still didn't work for me. There might be something more needed to make the vue components compile with the actual strings from the language files. 

![image](https://user-images.githubusercontent.com/4026726/199707744-5e32547d-b1d2-46a9-8392-6722e276a281.png)

**Reply by duncanmcclean** [reply]
Ah okay. Yeah, wrapping them in `{{` will probably do the job. Compiling addon JS is a little hard to do but if you PR the fix, I'll test it this evening & get it fixed!
